### ISSUE TITLE
Automatically convert weight and volume

### ISSUE THREAD

**Issue by alexouille123** [openfoodfacts/openfoodfacts-server#4573]
Automatically convert the multiplications by the number itself. Let me explain for example:  
![Quantity OFF](https://user-images.githubusercontent.com/67234740/100780511-187b8e80-340a-11eb-927f-deb5e2a1ce86.png)
Will automatically become :
![Quantity 2 OFF](https://user-images.githubusercontent.com/67234740/100780550-25987d80-340a-11eb-8881-976ff4fe5d9d.png)

**Comment by aleene** [comment]
Sounds good. Instead of using `(..)`, I propose to use comma's as separators. Now and then there are more than 2 values that need to be entered.

**Comment by alexouille123** [comment]
Where do you see `(..)` ?

**Comment by aleene** [comment]
`(`3 x 110 ml`)`

**Comment by alexouille123** [comment]
@aleene So, if I understand correctly, you'll want to make: 330 ml (3,110). That's a bit weird for understanding weight/quantity, isn't it?

**Comment by aleene** [comment]
No, no. I prefer the field to be like `330 ml, 3 x 110 ml, 6 oz, 200 g égoutté`, etc. Each item is the a synonym of the other and is equivalent (not in this example).

**Comment by Ban3** [comment]
FWIW [Guide for Producers](https://world.openfoodfacts.org/producers#format_des_donnees) guides to use parentheses:

> quantity: quantity, specify the unit, and possibly add in parentheses if the product contains several units (example: 120 g (4 x 30 g))

**Comment by alexouille123** [comment]
@aleene Ah okay, I see, thank you! 
@Ban3 That would mean choosing between 330 ml (3 x 110 ml) and 330 ml, 3 x 110 ml?
